prow/config.yaml

```diff
@@ -170,14 +170,22 @@ presubmits:
         env:
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /etc/service-account/service-account.json
+        - name: COVERALLS_TOKEN
+          value: /etc/coveralls-token/coveralls.txt
```

Comment:
oh actually I think this will be pointing `COVERALLS_TOKEN=/etc/coveralls-token/coveralls.txt` whereas goveralls expects `COVERALLS_TOKEN=$(cat /etc/coveralls-token/coveralls.txt)`...

Reply:
```
- name: COVERALLS_TOKEN
  valueFrom:
     secretKeyRef:
       name: k8s-multicluster-ingress-coveralls-token
       key: coveralls
```
instead and drop the volume stuff, I'll update the secret to match.

Reply:
I updated the secret, just replace this env entry here with the code above and drop the volume changes below.

Most of our secrets actually are files, but it looks like goveralls just wants the token value in the `env`.